components/Modals/SignUp.tsx

```diff
@@ -99,16 +99,24 @@ export default function LoginModal({ onClose, state }) {
 
         <HorizontalScrollView>
           {[1, 2, 3, 4, 5].map((i) => (
-            <Avatar
-              key={i}
-              avatar={`/assets/avatars/${i}.png`}
-              size={80}
-              onClick={() => setAvatar(i)}
-              className={clsx('inline-block h-20 w-20 rounded-full border-2', {
-                'border-white': avatar === i,
-                'border-transparent': avatar !== i,
-              })}
-            />
+            <button
```

Comment:
In case Kevin's comment wasn't clear this button needs a key as it is within the .map() function